src/components/map/MapComponet.jsx

```diff
@@ -0,0 +1,29 @@
+import { createRef, useEffect, useRef } from "react";
+import 'ol/ol.css'
+import { useMap } from "../../contexts/mapContext/mapContext";
+import DisplayLayers from "../layers/displayLayers";
+import DrawInteraction from "../layers/drawing/drawInteraction";
+import { Box } from "@mui/material";
+
+
+
+const MapComponent = () => {
+    const mapRef = useRef(useMap());
+    const ref = createRef()

```

Comment:
Please avoid using a generic name like _ref_. Instead, opt for a more specific name such as _mapContainerRef_.